administrator/components/com_admin/sql/updates/mysql/3.7.0-2016-06-14.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE if NOT EXISTS  `#__share_draft`(
+  `id` int(10) NOT NULL AUTO_INCREMENT COMMENT 'Primary Key to #__content',
```

Comment:
This is not correct. The primary key should not be related to anything. There should be an additional field called articleId which is the cross-reference to the ID of the article.